src/azul/health.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+
+from botocore.exceptions import ClientError
+
+from azul import config
+
+
+def get_elasticsearch_health():
+    try:
+        es = boto3.client('es')
+        elasticsearch = es.describe_elasticsearch_domain(DomainName=config.es_domain)
+
```

Comment:
I'd like to see an actual ping() using the ES client.

http://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch.ping